### ISSUE TITLE
New events are not being created

### ISSUE THREAD

**Issue by esossai** [GCalToolkit/Sheets2GCal#38]
Hi, when i add new rows to the sheet with the column UPDATE checked for some reason it does not create new events. All rows are new events and it only says events updated (which is not the case) and none created.

**Comment by esossai** [comment]
HI , Yes all the coluns are there and never touched. Never edited any of the headers or move any column around.

https://docs.google.com/spreadsheets/d/10hMaGsMlvdeAT6xspuBF0743UoXQrlL_iRnvmtP9V-8/edit#gid=0

Here is a copy of my sheets which is connected to my calendar, the data is real and nothing confidential there.

**Comment by GCalToolkit** [comment]
I made a copy of that Sheet, linked it to one of  my own calendars, deleted the Event ID data (and Guests ) and recreated all the existing events with no problems.  I then added a new row and  tried to update it - again, no problem. It seems that all the events in that Sheet have been successfully exported to  Google Calendar, so could you please add a row to that sheet which fails to export? That  way I can test with the exact same event data that you are using.

**Comment by esossai** [comment]
Could be the issue that i am always copying the columns from timezone onwards to all events i want to create? Because in this particular calendar the timezone, guests, etc is always the same info, i always copy the data to avoid having to enter all the guests one by one because they are always the same people

**Comment by GCalToolkit** [comment]
Yes, that's very likely to be the cause. The Event ID column is hidden by default so you'll have to click the little black double arrows just above row 1 to show it.

I don't know if copying data like that will also copy data in hidden columns... but the Event ID for a row must be empty so that Sheets2GCal knows that the row hasn't yet been exported Google Calendar.

Also, you can't just copy the guest data unfortunately because Sheets2GCal will currently only add guests if there's a comma separated list of email addresses, but I'll see if I can improve that.

**Comment by esossai** [comment]
Perfect. I will try to test that , probably it was copying all that data that was messing up the import. I will try and see if the copy was also messing up the event ID but probably yes. 

If you could improve to add guests im sure other users would like that too.

**Comment by GCalToolkit** [comment]
Ok, let me know how that goes!

**Comment by esossai** [comment]
It WORKED LIKE CHARM. It was the copy of data because the Event ID was right there in the middle of the data i was copying from one row to another. Maybe if the EVENT ID was hidden as the first column for instance this would not have happened. 

But i also tested creating new events and now i added one list of emails separated by comma on the guest column , and copy and past on the other rows and it also worked fine, created all the new events and added the guests. I just have to add the guests as a list like you siad. thank you.

**Comment by GCalToolkit** [comment]
Glad to hear that worked!

You can move the Event ID to column Z if you want; in fact you can move any column except "Delete" and "Update" which must be A and B.

You can also add your own columns, as long as they don't have the same names as those used by Sheets2GCal, and they aren't A or B.

**Comment by esossai** [comment]
Perfect, Thank you very much
